dubbo-test/dubbo-test-check/src/main/java/org/apache/dubbo/test/check/registrycenter/ZookeeperRegistryCenter.java

```diff
@@ -87,10 +98,47 @@ public ZookeeperRegistryCenter() {
      */
     private Map<OS, Map<Command, Processor>> processors = new HashMap<>();
 
+    /**
+     * The default unpacked directory.
+     */
+    private static final String UNPACKED_DIRECTORY = "apache-zookeeper-bin";
+
+    /**
+     * The target name of zookeeper binary file.
+     */
+    private static final String TARGET_ZOOKEEPER_FILE_NAME = UNPACKED_DIRECTORY + ".tar.gz";
+
+    /**
+     * The target directory.
+     * The zookeeper binary file named {@link #TARGET_ZOOKEEPER_FILE_NAME} will be saved in
+     * {@link #TARGET_DIRECTORY} if it downloaded successfully.
+     */
+    private static final String TARGET_DIRECTORY = "test" + File.separator + "zookeeper";
```

Comment:
It is better to  change the directory name similar to `.test`, `.tmp`.  The `test` directory can be used for meaningful things.

Reply:
> It is better to change the directory name similar to `.test`, `.tmp`. The `test` directory can be used for meaningful things.

OK, I have renamed the directory's name from test to .tmp